pertpy/preprocessing/_guide_rna.py

```diff
@@ -109,6 +112,93 @@ def assign_to_max_guide(
 
         return None
 
+    def assign_mixture_model(
+        self,
+        adata: AnnData,
+        model: str = "Poisson_Gauss_Mixture",
+        output_key: str = "assigned_guide",
+        no_grna_assigned_key: str = "Negative",
```

Comment:
```suggestion
        no_grna_assigned_key: str = "negative",
```

Maybe lowercase instead?